ci-operator/step-registry/interop/opp/odf-health/interop-opp-odf-health-commands.sh

```
@@ -0,0 +1,448 @@
+    ')" || true
+
+    if ! accessKey="$(oc get secret "${secretName}" -n "${ODF_NAMESPACE}" -o jsonpath='{.data.AWS_ACCESS_KEY_ID}' | base64 -d)"; then
+        AddResult "noobaa-s3-functional" "fail" "Cannot read NooBaa admin credentials"
+        return
+    fi
+    if ! secretKey="$(oc get secret "${secretName}" -n "${ODF_NAMESPACE}" -o jsonpath='{.data.AWS_SECRET_ACCESS_KEY}' | base64 -d)"; then
+        AddResult "noobaa-s3-functional" "fail" "Cannot read NooBaa admin credentials"
+        return
+    fi
+
+    typeset bucket="odf-health-check-$$"
```

Shellcheck (0.11.0)

[warning] bucket appears unused. Verify use (or export if used externally).

(SC2034)